pal/GCC/ARM_CM4F/az_ulib_port.h

```diff
@@ -87,6 +87,18 @@ extern "C"
     return result;
   }
 
+  __attribute__((always_inline)) static inline void AZ_ULIB_PORT_GET_DATA_CONTEXT(
+      volatile void** data_address)
+  {
+    __asm inline("       str     r9, [%0]                " : "=&r"(data_address) : : "memory");
+  }
+
+  __attribute__((always_inline)) static inline void AZ_ULIB_PORT_SET_DATA_CONTEXT(
+      volatile void* data_address)
```

Comment:
are these only implemented for cortex m4?

Reply:
Yes, Windows and Linux handle relocation in a different way, using the MMU.